internal/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -2645,7 +2640,7 @@ func newMachineHealthCheck(namespace, clusterName string) *clusterv1.MachineHeal
 				},
 			},
 			MaxUnhealthy:       &maxUnhealthy,
-			NodeStartupTimeout: &metav1.Duration{Duration: 1 * time.Millisecond},
+			NodeStartupTimeout: &metav1.Duration{Duration: 5 * time.Second},
```

Comment:
It's really hard for me to tell what impact this has, this func has ~ 20-30 usages. I would prefer if we only change this locally

Reply:
@cahillsf Probably not intentional but these are still different :)

Reply:
ah yes, i actually made an additional unintended change instead of reverting 🙃